tests/clients/test_job.py

```diff
@@ -84,7 +99,47 @@
 
 JOB_SUMMARY = JobSummary(id=JOB_ID, metadata=JOB_METADATA)
 JOB_SUMMARY_BODY = {"jobId": str(JOB_ID), "meta": JOB_METADATA_BODY}
-
+INSTANCE_SIZE = InstanceSize(milli_cpus=MILLI_CPUS, memory_mb=MEMORY_MB)
+INSTANCE_SIZE_BODY = {"milliCpus": str(MILLI_CPUS), "memoryMb": str(MEMORY_MB)}
+JOB_PARAMETERS = JobParameter(
```

Comment:
Since this is a single parameter:
```suggestion
JOB_PARAMETERS = JobParameter(
```